modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormViewFormInstanceRecordsDisplayContext.java

```diff
@@ -575,6 +596,38 @@ protected void setDDMFormInstanceRecordSearchTotal(
 		ddmFormInstanceRecordSearch.setTotal(total);
 	}
 
+	private String _getColumnValueFieldTypeSelect(
+		DDMFormFieldOptions formFieldOptions,
+		List<String> renderedFormFieldValues) {
+
+		Stream<String> stream = renderedFormFieldValues.stream();
+
+		List<String> convertedFormFieldValues = stream.flatMap(
+			s -> Arrays.stream(StringUtil.split(s, CharPool.COMMA))
+		).map(
+			String::trim
+		).collect(
+			Collectors.toCollection(LinkedList::new)
```

Comment:
Hey @carolmariaabb,

Is there any specific reason to collect the stream output as a `LinkedList` instead of a usual list using `Collectors.toList()`, for example?